src/SprykerEco/Shared/Computop/Transfer/computop.transfer.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
+<transfers xmlns="spryker:transfer-01"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="spryker:transfer-01 http://static.spryker.com/transfer-01.xsd">
+
+    <transfer name="ComputopCreditCardPayment">
+        <property name="payId" type="string" />
+        <property name="merchantId" type="string" />
+        <property name="transId" type="string" />
+        <property name="amount" type="int" />
+        <property name="currency" type="string" />
+        <property name="mac" type="string" />
+        <property name="urlSuccess" type="string" />
+        <property name="urlFailure" type="string" />
+        <property name="response" type="string" />
+        <property name="urlNotify" type="string" />
+        <property name="userData" type="string" />
+        <property name="capture" type="string" />
+        <property name="reqId" type="string" />
+        <property name="plain" type="string" />
+        <property name="custom" type="string" />
+        <property name="expirationTime" type="string" />
+        <property name="orderDesc" type="string" />
+        <property name="customField" type="array" />
+        <property name="paymentMethod" type="string" />
+
+        <property name="data" type="string" />
+        <property name="len" type="int" />
+        <property name="response" type="string" />
+        <property name="txType" type="string" />
```

Comment:
:+1: Should len then also stay as it is for consistency in this specific transfer?